examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/EasyJsonObjectRequest.java

```diff
@@ -0,0 +1,44 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.util.Log;
+
+import com.android.volley.NetworkResponse;
+import com.android.volley.ParseError;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.HttpHeaderParser;
+import com.android.volley.toolbox.JsonObjectRequest;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.UnsupportedEncodingException;
+
+public class EasyJsonObjectRequest extends JsonObjectRequest {
+
+    private static final String TAG = "EasyJsonObjectRequest";
```

Comment:
Im just saying that this String will not be caught by a refactor. So if we decided that "EasyJsonObjectRequest" was not a descriptive enough class name and refactored it to "ChimneySwapJsonRequest", an IDE will leave that string as "EasyJsonObjectRequest".